read-historical-data.md

```diff
@@ -152,3 +152,17 @@ Pay special attention to the following:
     > **Note:**
     >
     > You should use `@@` instead of `@` before `tidb_snapshot` because `@@` is used to denote the system variable while `@` is used to denote the user variable.
+
+## Restoring old versions
+
+The first thing you probably want to do when you need to restore from an older version is to make sure GC doesn't remove it while you are working on it. This can be done by setting the GC as is done in the example below. Don't forget to set it back to the previous value after the restore.
+
+```sql
+SET GLOBAL tidb_gc_life_time="60m";
```

Comment:
Note for other reviewers: this syntax is new to TiBB 5.0 (but that's OK, this PR does not target older versions).